public/login.js

```diff
@@ -31,23 +50,33 @@ function login(event) {
   const urlParams = new URLSearchParams(window.location.search)
   const identityProviderUrl =
     urlParams.get('identity_provider_url') || 'https://api.core.ac.uk/'
+  const fallbackIDP =
+    urlParams.get('fallback_idp') || 'https://dashboard.core.ac.uk/'
```

Comment:
I guess this url param is never set so why to check it?

```
  const fallbackIDP = 'https://dashboard.core.ac.uk/'
```

Reply:
I forget to set it but it works fine with the default value 🙂